### PR TITLE
Docs/theme switcher in demo

### PR DESCRIPTION
**Pull request by adamkudrna**
### Additional context

obrazek
obrazek

### Issue reference

https://jira.almacareer.tech/browse/DS-1479

### CODE REVIEW

File: apps/demo/config/vite/app.ts [hunk 1]
```diff
@@ -41,6 +41,16 @@ export default defineConfig({
       },
       runtimeOptions: {
         data: {
+          themes: [
+            {
+              id: 'light-default',
```
Comment (crishpeen) [inline apps/demo/config/vite/app.ts:46]: maybe we can get this from the JS tokens? It is not there yet, but the exporter will return it in the future.

File: apps/demo/config/vite/app.ts [hunk 2]
```diff
@@ -41,6 +41,16 @@ export default defineConfig({
...
+              name: 'Light default',
```
Comment (crishpeen) [inline apps/demo/config/vite/app.ts:47]: title case?

File: apps/web-twig-demo/templates/partials/themeSwitcher.html.twig [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{% set inputProps = { autocomplete: 'off' } %}
+
+<Select
+  inputProps={ inputProps }
+  id="select-theme"
+  label="Theme"
+  name="theme"
+  onchange="switchTheme(event)"
+  isFluid
+  isLabelHidden={{ isLabelHidden is defined ? isLabelHidden : false }} {# The `default()` filter doesn't work. #}
```
Comment (crishpeen) [inline apps/web-twig-demo/templates/partials/themeSwitcher.html.twig:10]: why? it works in components 🤔

**Review by crishpeen — APPROVED** [review]

**Comment by crishpeen** [comment]
We might have a bug there. On first load the select gets weird hover background. When I change the value, it does not happen anymore.

File: apps/demo/config/vite/app.ts [hunk 4]
```diff
@@ -41,6 +41,16 @@ export default defineConfig({
...
+            },
+            {
+              id: 'light-on-brand',
+              name: 'Light on brand',
```
Comment (literat) [inline apps/demo/config/vite/app.ts:51]: Title Case also here.

**Pull request merged**